activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/assertions/ProcessAssertions.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2017-2020 Alfresco.
```

Comment:
in Activiti we have `Alfresco Software, Ltd.`, shouldn't it be the same here?

Reply:
yes it should, let me check where that comes from